doc/maintaining/configuration.rst

```diff
@@ -1199,6 +1199,19 @@ Default value: |config:solr_timeout|
 
 The option defines the timeout in seconds until giving up on a request. Raising this value might help you if you encounter a timeout exception.
 
+ckan.search.remove_deleted_packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Example::
+
+ ckan.search.remove_deleted_packages = true
+
+Default value: |config:ckan.search.remove_deleted_packages|
+
+Removes deleted datasets from search index (default behaviour).  
+If `False`, you can use `include_deleted` in `package_search` 
+and deleted packages will be visible in the packages list.
```

Comment:
```suggestion
By default, deleted datasets are removed from the search index so are no longer available in searches. To keep them in the search index, set this setting to ``False``. This will enable the  ``include_deleted`` parameter in the  :py:func:`ckan.logic.action.get.package_search` API action.
```